pyrasterframes/src/main/python/docs/time-series.pymd

```diff
@@ -5,6 +5,8 @@
 ```python setup, echo=False
 from IPython.display import display
 
+from docs import folium_map
```

Comment:
Is this something we could perhaps handle in `rf_ipython` through some monkey patching?